projects/ngx-formentry/src/components/ngx-datetime-picker/ngx-datetime-picker.html

```diff
@@ -18,7 +18,7 @@
           [value]="value"
           [owlDateTime]="dt1"
           [ofeOwlDateTimeTrigger]="dt1"
-          placeholder="mm/dd/yyyy"
+          placeholder="{{getPlaceholderValue()}}"
```

Comment:
`{{`?

Reply:
@ibacher, did you mean to refer to the interpolation syntax?

Reply:
Yes… I forgot this was Angular.